test/exercism/guest_test.rb

```diff
@@ -1,8 +1,51 @@
 require_relative '../test_helper'
+require_relative '../active_record_helper'
```

Comment:
Well not *directly* but it's only using UserExercise because that is what User.exercise returns.